### PR TITLE
Modernize Css build pipeline

### PR DESCRIPTION
**Pull request by deeptibaghel**
Fixes https://github.com/mozilla/fxa-content-server/issues/6165

@vladikoff with these changes if i run:
node node_modules/webpack/bin/webpack.js 
it watches for changes in scss files and updates main.css in dist/bundle/styles folder
I followed this: https://florianbrinkmann.com/en/4240/sass-webpack/

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 44ea1d1fa67d50e72b432d6b9cde42b9ad90ebd2
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: cfc3a2cac2212c207f8bbc4aa8f7373cf8602206
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: webpack.config.js [hunk 1]
```diff
@@ -32,9 +32,11 @@ const webpackConfig = {
       'uuid',
       'vat',
       'webrtc',
+      'styles/main.scss'
     ],
     head: './head/boot.js'
-  },
+   },
+   watch:true,
```
Comment (vladikoff) [inline webpack.config.js]: add a space after `:`

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 44ea1d1fa67d50e72b432d6b9cde42b9ad90ebd2
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: cfc3a2cac2212c207f8bbc4aa8f7373cf8602206
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by deeptibaghel** [comment]
@vladikoff , hash is now added to main.css in production mode and the file is generated in folder app/styles. How do i make webpack build in dev mode ?  Travis is getting stuck though, not sure why.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 44ea1d1fa67d50e72b432d6b9cde42b9ad90ebd2
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: cfc3a2cac2212c207f8bbc4aa8f7373cf8602206
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 10c29e2b93442a077202d0c6d346df41a23fe2a6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 440973ff78a409f19c284ff2c00fa05de60770b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "autoprefixer": "^9.0.1",
```
Comment (vladikoff) [inline package.json]: * remove the `^` from all new dependencies
* I think we can remove:

 "grunt-sass": "2.0.0", and ""grunt-autoprefixer": "3.0.4","

File: grunttasks/server.js [hunk 3]
```diff
@@ -13,7 +13,6 @@ module.exports = function (grunt) {
       'selectconfig:app',
       'l10n-create-json',
       'l10n-generate-tos-pp:app',
-      'css',
       'copy:styles',
```
Comment (vladikoff) [inline grunttasks/server.js]:
```diff
diff --git a/grunttasks/server.js b/grunttasks/server.js
index 8f8f48c1..1b66e2e6 100644
--- a/grunttasks/server.js
+++ b/grunttasks/server.js
@@ -13,7 +13,6 @@ module.exports = function (grunt) {
       'selectconfig:app',
       'l10n-create-json',
       'l10n-generate-tos-pp:app',
-      'copy:styles',
       'serverproc:app'
     ]);
   });
```

We can also remove the copy:styles

**Comment by vladikoff** [comment]
- [ ] Fixes from above
- [ ] Update docs to remove the `sass` task and combine the "changes to stylesheets" section.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 10c29e2b93442a077202d0c6d346df41a23fe2a6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 440973ff78a409f19c284ff2c00fa05de60770b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: server/bin/fxa-content-server.js [hunk 4]
```diff
@@ -82,7 +82,8 @@ function makeApp() {
     const webpackCompiler = webpack(webpackConfig);
 
     app.use(webpackMiddleware(webpackCompiler, {
-      publicPath: '/bundle/'
+      publicPath: '/bundle/',
+      writeToDisk: true
```
[the remark this answers is not in the document]
Comment (deeptibaghel) [reply server/bin/fxa-content-server.js]: @vladikoff, i tried to use that function here but it throws compilation error

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 10c29e2b93442a077202d0c6d346df41a23fe2a6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 440973ff78a409f19c284ff2c00fa05de60770b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 10c29e2b93442a077202d0c6d346df41a23fe2a6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 440973ff78a409f19c284ff2c00fa05de60770b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 852aa9e8ab18e247892d6b5745785a5db0421138
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 10c29e2b93442a077202d0c6d346df41a23fe2a6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 440973ff78a409f19c284ff2c00fa05de60770b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 852aa9e8ab18e247892d6b5745785a5db0421138
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 10c29e2b93442a077202d0c6d346df41a23fe2a6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 440973ff78a409f19c284ff2c00fa05de60770b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 852aa9e8ab18e247892d6b5745785a5db0421138
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: b6eaf8f76947efcb1e53f38b180f1c0d87763925
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Pull request merged**